src/main/java/edu/kis/powp/jobs2d/drivers/builders/FigureScript1Builder.java

```diff
@@ -0,0 +1,35 @@
+package edu.kis.powp.jobs2d.drivers.builders;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.drivers.commands.ComplexCommand;
+import edu.kis.powp.jobs2d.drivers.commands.DriverCommand;
+import edu.kis.powp.jobs2d.drivers.commands.OperateToCommand;
+import edu.kis.powp.jobs2d.drivers.commands.SetPositionCommand;
+import edu.kis.powp.jobs2d.magicpresets.FiguresJoe;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class FigureScript1Builder implements Job2dDriver, Builder{
+    private final List<DriverCommand> commandList = new ArrayList<>();
+
+    @Override
+    public void setPosition(int x, int y) {
+        this.commandList.add(new SetPositionCommand(x, y));
+    }
+
+    @Override
+    public void operateTo(int x, int y) {
+        this.commandList.add(new OperateToCommand(x, y));
+    }
+
+    @Override
+    public Object build() {
+        if (this.commandList.size() == 0) {
+            /* we could have also used another class for implementing Job2Driver (adapter)
+               but I believe this solution is more elegant as this builder has very specialized functionality */
```

Comment:
Actually, it would be very easy to convert those Builders to a single generic functionality (to cover building command from any script). Actually it is even possible now, because we can externally execute any set of commands on FigureScript1Builder (but the name doesn't suggest such possibility).